src/YouTrackSharp/AgileBoards/AgileSettings.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace YouTrackSharp.AgileBoards
+{
+    public class AgileSettings
+    {
+        public AgileSettings()
+        {
+            Projects = new List<Project>();
+        }
+
+        [JsonProperty("projects")]
+        public ICollection<Project> Projects { get; set; }
+
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        [JsonProperty("query")]
+        public string Query { get; set; }
+
+        [JsonProperty("columnSettings")]
+        public ColumnSettings ColumnSettings { get; set; }
+
+        [JsonProperty("timeField")]
+        public Field TimeField { get; set; }
+
+        [JsonProperty("projectsColors")]
+        public object ProjectsColors { get; set; }
```

Comment:
nit: rename the C# property to `ProjectColors`

Reply:
This seems to be an error in the API. As far as I can tell the property is not included in the xml return, only in json. I removed the property from the class untill further notice.